extendr-macros/src/dataframe.rs

```diff
@@ -0,0 +1,58 @@
+use proc_macro::TokenStream;
+use quote::quote;
+use syn::{parse_macro_input, Data, DataStruct, DeriveInput};
+
+fn parse_struct(input: &DeriveInput, datastruct: &DataStruct) -> TokenStream {
+    #![allow(non_snake_case)]
```

Comment:
Why do you need non snake case here. Can't you just rename `A`?

Reply:
Removed.